### DISCUSSION TITLE
Host Live stream

### DISCUSSION THREAD

**Discussion by DGMurdockIII** [godotengine/godot-proposals#10215] [Platforms]
I have a idea to that you guys should host monthly live stream eather on discord or YouTube, twitch or kick to tell people what's new with engine and possibly showcase new games that have come out made with the engine

**Comment by AThousandShips** [comment]
Monthly streams would probably take up a lot of time and work though, and we don't necessarily have major news every month that can be shared easily, the process and time for writing our regular dev blogposts already take some time and effort and a livestream would be a lot more work I feel

**Reply by DGMurdockIII** [reply]
OK dosent have to monthly maybe once in a while
